media/css/firefox/tracking-protection-tour.less

```diff
@@ -228,20 +232,23 @@ main > header {
         button {
             .font-size(13px);
             background-color: #6cc042;
-            border-radius: 5px;
+            border-radius: 3px;
             border: 1px solid #c0c0c0;
             color: #fff;
-            padding: 5px 2.5em;
+            padding: 4px 30px;
             margin-left: 30px;
-            transition: background-color .1s ease-in-out;
+            transition-property: background-color, border-color;
+            transition-duration: 150ms;
 
             &:hover,
             &:focus {
+                border-color: #5f963d;
                 background-color: #6bac44;
             }
         }
     }
 
+    // https://mxr.mozilla.org/mozilla-central/source/toolkit/themes/windows/global/icons/close@2x.png
```

Comment:
Guessing these `mxr.` comments are kept as a reference to the original source images?